index.js

```diff
@@ -135,6 +136,10 @@ class ClinicBubbleprof extends events.EventEmitter {
     })
     b.add(scriptPath)
     const scriptFile = b.bundle()
+
+    if (/\bnode-clinic\b/i.test(process.env.NODE_DEBUG)) {
+      scriptFile.pipe(minifyStream({ sourceMap: false, mangle: false }))
+    }
```

Comment:
I would prefer a `settings.debug` option, so that we can do `clinic bubble --debug`  on the main project.